.env.example

```
@@ -64,3 +64,9 @@ MAILSANGJA_FCM_SERVICE_ACCOUNT_KEY_JSON=
 MAILSANGJA_FCM_SERVICE_ACCOUNT_KEY_PATH=firebase-key.json
 MAILSANGJA_FCM_LOGO_IMAGE_URL=https://www.ajou.ac.kr/_res/ajou/kr/img/intro/img-symbol.png
 MAILSANGJA_FCM_THREAD_DETAIL_URL_TEMPLATE=https://mail.ajou.app/mail/inbox?thread={threadId}&message={messageId}
+
+PORTONE_API_SECRET=your-portone-api-secret
+PORTONE_PAYMENT_QUERY_URI=https://api.portone.io/payments
+PORTONE_CONNECT_TIMEOUT=3s
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The PORTONE_CONNECT_TIMEOUT key should go before the PORTONE_PAYMENT_QUERY_URI key

(UnorderedKey)